convert/tsdb.go

```diff
@@ -192,9 +121,10 @@ func (rr *indexRowReader) ReadRows(buf []parquet.Row) (int, error) {
 		chkLbls.Range(func(l labels.Label) {
 			colName := schema.LabelNameToColumn(l.Name)
 			lc, _ := rr.schema.Lookup(colName)
```

Comment:
lc is not needed anymore i think, we can use labelsIndex ( i think we should call labelsIndex labelsColumn instaed  )